nixos/modules/flyingcircus/packages/sensu/default.nix

```diff
@@ -1,10 +1,8 @@
-{ lib, bundlerEnv, ruby_2_0, python2, pkgs, which, defaultGemConfig, zlib, libxml2, graphicsmagick, pkgconfig, imagemagickBig }:
+{ lib, bundlerEnv, ruby, python2, pkgs, which, defaultGemConfig, zlib, libxml2, graphicsmagick, pkgconfig, imagemagickBig }:
 
 bundlerEnv {
   name = "sensu-0.22.1";
 
-  ruby = ruby_2_0;
```

Comment:
While trying this stuff, I found that sensu gets built with Ruby 2.2, not 2.1. I guess that this is not intended according to the callPackage statement earlier on. Is there a "inherit ruby" missing?